### DISCUSSION TITLE
HmIP-HAP-B1 in Home Assistent didn´t work correctly

### DISCUSSION THREAD

**Discussion by Gandalf1-1** [OpenCCU/OpenCCU#2637] [🇺🇸 Q&A] [answered]
I hope you don't mind me posting my problem here.
I now use the rasperrymatic integration within home assistant. 
Before I imported the backup, I saw my CCU and both HAPs on the start page. 
Now I have uploaded my old backup. 
Initially, I can also see my two HmIP-HAP-B1s listed on the start page as usual.
After each restart, they disappear after a few minutes.  They are then listed on the start page under "Alarm messages" with "RF-Gateway HmIP_IP_Gateway_HzRaum (........) not connected". 
I then deleted one HAP from the Rasperymatic in order to learn it again.
However, it then remains in the device list, "delete" is greyed out and it can therefore not be re-registered even after a device reset.  
(after the restart it is dutycycle and carrier Sense is always 0 for the HAPs)

Can anyone help me further
Thanks for your help
Reiner
![Screenshot 2024-02-01 112534](https://github.com/jens-maus/RaspberryMatic/assets/118777380/ecf7cbac-8539-4b4f-9eb2-8681dc5eeb7a)
![Screenshot 2024-02-01 112832](https://github.com/jens-maus/RaspberryMatic/assets/118777380/249669fe-eed8-4819-a8b0-3d96a008ec0c)

**Comment by jens-maus** [comment, marked as answer]
Please read the "Known limitations" section of the HomeAssistant Addon documentation closely:

https://github.com/jens-maus/RaspberryMatic/wiki/Installation-HomeAssistant#known-limitations

There you will see that currently using any `HmIP-HAP` or `HmIPW-DRAP` with a RaspberryMatic HA Addon requires a manual workaround, thus is not directly supported due to limitations of HomeAssistant.

**Comment by Gandalf1-1** [comment]
Thank you very much for your quick reply. I had overlooked that.
Unfortunately, I haven't really got any further success.
When I am at the end with the terminal (see picture for steps) and switch to the Rasperrymatic, I get several error messages "HmIP-RF , BidCos-RF .... " after clicking away these messages Rasperrymatic seems to restart. Surface is as before 1 hap with Dutycyle 0 and the other still not removeable. 
Have done it 3 times always the same result.  
Communication with fhem only worked again after a complete restart of the home assistant.
![Screenshot 2024-02-01 150520](https://github.com/jens-maus/RaspberryMatic/assets/118777380/105fb1fe-1323-480a-9f98-6e02922a015a)
